android/src/main/java/com/mr/flutter/plugin/filepicker/FilePickerPlugin.java

```diff
@@ -117,7 +123,7 @@ public boolean onRequestPermissionsResult(int requestCode, String[] strings, int
 
   @Override
   public void onMethodCall(MethodCall call, Result result) {
-    this.result = result;
+    FilePickerPlugin.result = result;
```

Comment:
@Dancovich why this change? Do you find it more readable?

Reply:
In Java static fields are supposed to be accessed through their classes. The original call is actually translated to `FilePickerPlugin.result` by the compiler and lint gives a warning about that.

One thing I noticed is that the `result` field is static in this class. Is there a particular reason for that? It seems to me Flutter will create a new instance of the plugin when it calls `FilePicker.registerWith` and I assumed it would create one instance of the plugin for the entire app life cycle, so the `result` field could be an instance field to `FilePickerPlugin`.

Anyway, since this was way out of scope for this bug I just changed the instance access to a class access like lint suggests, but this change won't actually affect the code in anyway if it's reverted.

Reply:
Yes, I know that. I only questioned because I was wondering if you changed it to be more accordingly to Java practices or if it somehow was related to the purposed fix, which would surprise me. 😁

Anyway, could you please change the target branch to `beta` instead? I usually merge to master only after testing and when I’m about to publish on pub.dev as well.

Thank you!